lib/compat/wordpress-6.8/rest-api.php

```diff
@@ -0,0 +1,81 @@
+<?php
```

Comment:
Maybe this should be renamed to `block-comments.php`? It's about more than just changes in the REST API.